install/create_certs.sh

```diff
@@ -0,0 +1,22 @@
+
+
+web_ingress_public_ip=$(jq -r '.nodes[] | select(has("web_ingress")) | .web_ingress.public_ip' config.json)
+web_ingress_ssh_username=$(jq -r '.nodes[] | select(has("web_ingress")) | .web_ingress.ssh_username' config.json)
+
+web_ingress_ssh_command="ssh -o StrictHostKeyChecking=no $web_ingress_ssh_username@$web_ingress_public_ip"
+
+certs_dir="/opt/neuraldb_enterprise/certs"
+
+$shared_file_system_ssh_command <<EOF
+    sudo apt install openssl
+    sudo mkdir -p $certs_dir
+    cd $certs_dir
+    openssl req -x509 -newkey rsa:4096 -keyout traefik.key -out traefik.crt -days 365 -nodes -subj "/CN=$web_ingress_public_ip"
```

Comment:
i think this `-nodes` will not encrypt the private key right, do you think we should use encrypt and decrypt for security purposes?

Reply:
I don't think -nodes is necessary, it's for putting a passphrase on the file. It might add bit more complexity for the user than needed, so I'll leave it as is for now.

Also the files are created directly on their VMs and not locally, so that should provide enough protection. And your above permissions suggestion would make it even more secure.